rule/exported.go

```diff
@@ -70,7 +77,12 @@ func (*ExportedRule) Name() string {
 	return "exported"
 }
 
-func (r *ExportedRule) getConf(args lint.Arguments) (checkPrivateReceivers, disableStutteringCheck, sayRepetitiveInsteadOfStutters bool) {
+func (r *ExportedRule) getConf(args lint.Arguments) (
+	checkPrivateReceivers,
+	disableStutteringCheck,
+	sayRepetitiveInsteadOfStutters bool,
+	checkPublicInterface bool,
+) {
```

Comment:
This seems to be a good candidate for refactoring. Instead of adding another bool value, I'd refactor it to a struct.